docs/conf/portland/2025/writing-day.md

```diff
@@ -35,6 +35,12 @@ If you have a project, we encourage you to submit it before the conference! This
 
 Find specific examples on the [Portland Writing Day 2023 project list](https://www.writethedocs.org/conf/portland/2023/writing-day/#project-listing).
 
+<div class="announcement" style="background-color:white;">
+    <div class="uk-container">
+    <a style="border-bottom: none; font-size: .875rem;" class="uk-button uk-button-announcement uk-text-center" href="https://forms.gle/aCsZqb6peUUtxHJ49">Submit your Writing Day project</a>
```

Comment:
If you add the url to https://github.com/writethedocs/www/blob/main/docs/_data/portland-2025-config.yaml#L171

you can then use `{{ writing_day.url }}` or something in the form.

Reply:
Not enough brain for this right now, happy to have it be a fast follow.